eng/pipelines/coreclr/jitrollingbuild.yml

```diff
@@ -33,7 +33,8 @@ extends:
           - windows_x64
           - windows_x86
           - windows_arm64
-          disableComponentGovernance: true # Not a shipping artifact
```

Comment:
should we also do similar change in superpmi-* pipelines too?

Reply:
The only other one it matters for is superpmi-collect (the only other pipeline that runs in the `internal` instance), and that already sets it.